tools/dts/morello-qemu-gicv2.dts

```diff
@@ -0,0 +1,376 @@
+/dts-v1/;
```

Comment:
Where does this device tree come from, is it extracted from QEMU?

Reply:
Yes, it's from QEMU. I understand it could be generated at build time, but just wanted to fix the QEMU dts/config here

Reply:
That is fine, just leave a comment then what was the exact command line and QEMU version, so there is some reference. I wonder, is this Morello/QEMU is some custom build or are there some releases also?